### PR TITLE
Changed "RemoteDebuggingVersion" from enum to string.

### PR DESCRIPTION
**Pull request by haocs**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@haocs__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hach). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@haocs you will need to bump up the version of the nugget package to be able to publish the new one.

**Comment by hovsepm** [comment]
@haocs make sure that you will build locally after version change. Build will automatically update AssemblyInfo.cs file.

**Comment by markcowl** [comment]
@haocs was there a change to hydra-specs-pr as well?

File: src/ServiceManagement/WebSite/WebSites.Tests/ScenarioTests/WebSites.Config.ScenarioTests.cs [hunk 1]
```diff
@@ -95,6 +95,8 @@ public void GetWebSiteConfig()
             Assert.NotNull(siteAuthSettings);
             Assert.Equal("00000000-0000-0000-0000-7984e05b758c", siteAuthSettings.AADClientId);
             Assert.Equal("https://sts.windows.net/00000000-0000-0000-0000-19d76fef90d7/", siteAuthSettings.OpenIdIssuer);
+
```
Comment (markcowl) [inline src/ServiceManagement/WebSite/WebSites.Tests/ScenarioTests/WebSites.Config.ScenarioTests.cs]: remove blank line

**Comment by markcowl** [comment]
@haocs It looks like the KnownValues class was not included - did this not get copied over?

**Comment by haocs** [comment]
@markcowl yes. https://github.com/Azure/hydra-specs-pr/pull/1242

**Comment by haocs** [comment]
new PR to hydra-spec-pr referenced https://github.com/Azure/hydra-specs-pr/pull/1243

**Comment by haocs** [comment]
@markcowl @hovsepm I've updated this PR regarding your comments. Please give it a review.

**Pull request merged**